package.json

```diff
@@ -27,6 +27,7 @@
 		"@react-navigation/bottom-tabs": "^6.3.2",
 		"@react-navigation/drawer": "^6.4.3",
 		"@react-navigation/native": "^6.0.11",
+		"@react-navigation/native-stack": "^6.7.0",
```

Comment:
Not sure why this one was added. You could use 

`createStackNavigator` that we use in `UserAuthNavigation` instead, so we don't have to install more libs 🙏